src/guide/scaling-up/state-management.md

```diff
@@ -62,26 +62,26 @@ Isto é uma representação simples do conceito do "fluxo de dados de uma via":
   <img alt="diagrama do fluxo de estado" src="./images/state-flow.png" width="252px" style="margin: 40px auto">
 </p>
 
-No entanto, a simplicidade começa a decompor-se quando temos **vários componentes que partilham um estado comum**:
+No entanto, a simplicidade começa a falhar quando temos **vários componentes que partilham um estado comum**:
 
-1. Várias visões que podem depender do mesmo pedaço de estado.
-2. Ações de visões diferentes que podem precisar mudar o mesmo pedaço de estado.
+1. Várias visões que podem depender da mesma parte do estado.
+2. Ações de visões diferentes que podem precisar mudar a mesma parte do estado.
 
-Para o primeiro caso, uma possível solução é "transportando" o estado partilhado para cima para um componente ancestral comum, e depois passar para baixo como propriedades. No entanto, isto torna-se rapidamente entediante em árvores de componente com hierarquias profundas, levando para um outro problema conhecido como [Perfuração de Propriedade](/guide/components/provide-inject#prop-drilling).
+Para o primeiro caso, uma possível solução é "elevando" o estado partilhado para cima para um componente ancestral comum, e depois passar para baixo como propriedades. No entanto, isto torna-se rapidamente entediante em árvores de componente com hierarquias profundas, levando para um outro problema conhecido como [Perfuração de Propriedade](/guide/components/provide-inject#prop-drilling).
```

Comment:
Em inglês:
_For case one, a possible workaround is by "lifting" the shared state up to a common ancestor component, and then pass it down as props. However, this quickly gets tedious in component trees with deep hierarchies, leading to another problem known as [Prop Drilling](/guide/components/provide-inject#prop-drilling)._

Sugestões:
```suggestion
Para o primeiro caso, uma possível solução é "elevar" o estado partilhado até um componente ancestral comum, e depois passá-lo para baixo como propriedades. No entanto, isso se torna entediante rapidamente em árvores de componente com hierarquias profundas, levando a outro problema conhecido como [Perfuração de Propriedade](/guide/components/provide-inject#prop-drilling).
```